capnslog/pkg_logger.go

```diff
@@ -27,7 +29,10 @@ type PackageLogger struct {
 const calldepth = 2
 
 func (p *PackageLogger) internalLog(depth int, inLevel LogLevel, entries ...interface{}) {
-	if inLevel != CRITICAL && p.level < inLevel {
+	p.mu.Lock()
+	mylevel := p.level
+	p.mu.Unlock()
+	if inLevel != CRITICAL && mylevel < inLevel {
 		return
 	}
 	logger.Lock()
```

Comment:
Is there any reason not to just piggyback on logger.Lock() for serializing the accesses to PackageLogger.LogLevel?
I don't think we need to worry about lock contention here